src/ShadowDOM/wrappers/Range.js

```diff
@@ -20,23 +20,69 @@
 
   var OriginalRange = window.Range;
 
+  var ShadowRoot = scope.wrappers.ShadowRoot;
+
+  function isElement(node){
```

Comment:
this utility is very good. The Range.js now has no style errors according to jscs. I also fixed the styles of some of the code I added before. SD has 274 google code style errors according to jscs. Maybe every time a file gets changed the developer who changes it could fix the styles.